core/src/main/scala/kafka/tools/ReplicaVerificationTool.scala

```diff
@@ -43,7 +43,7 @@ import org.apache.kafka.common.utils.{LogContext, Time}
 import org.apache.kafka.common.{Node, TopicPartition}
 
 import scala.collection.JavaConverters._
-import org.apache.kafka.common.config.ClientDnsLookup
+import org.apache.kafka.clients.ClientDnsLookup
```

Comment:
There is a wildcard import in this class for org.apache.kafka.clients, so this import is not required.